generics/middleware/authenticator.js

```diff
@@ -98,7 +98,7 @@ module.exports = async function (req, res, next) {
     return
   }
 
-  if (req.path.includes("keywords")) {
+  if (req.path.includes("keywords") || req.path.includes("uploadFile1")) {
```

Comment:
@rakeshSgr - What is uploadFile1 ? aws/uploadFile, azureUploadFile needs to be there

Reply:
resolved